### DISCUSSION TITLE
Borders Not showing

### DISCUSSION THREAD

**Discussion by AKA-Akhil** [end-4/dots-hyprland#558] [Q&A]
I tried editing these in the general general.conf

border_size = 3
col.active_border = 0xffff0000
col.inactive_border = 0xff6f6f6f

but the changes only took place for border size how do change the border colors

**Comment by end-4** [comment]
looks right to me
https://wiki.hyprland.org/Configuring/Variables/#general

**Reply by AKA-Akhil** [reply]
this is the thing

https://github.com/end-4/dots-hyprland/assets/145592032/f7c80861-83c2-453d-9e39-272592981c37

**Reply by end-4** [reply]
oh
0xffff0000 is gonna be transparent since the two last digits are for transparency

**Reply by AKA-Akhil** [reply]
but isnt the first ff representing the opacity

**Reply by end-4** [reply]
ughhhhhhh okay hyprland is weird
what if you specify the color in another format like rgba()

**Reply by AKA-Akhil** [reply]
Still its giving me the same issue. The border color remains the unchanged.

**Reply by mrchipsers** [reply]
no fix?

**Reply by D8epDr8am** [reply]
Try to comment the border color lines from general.conf:


# col.active_border = rgba(0DB7D4FF)
# col.inactive_border = rgba(31313600)
Then edit colors.conf

**Reply by mrchipsers** [reply]
thanks that worked

**Comment by Prak005** [comment]
Is this still not fixed or am I doing it wrong? Same issue can change the border sizes but can't change their color
